www-apps/cgit/cgit-1.2.3.ebuild

```diff
@@ -68,15 +68,23 @@ src_prepare() {
 	epatch_user
 }
 
+src_configure() {
```

Comment:
This doesn't seem quite right to me. Doing this in `src_configure` clobbers the default `src_configure`, maybe it would be better to just throw in the one option at the `emake` command line?

Reply:
It's been more than two years, I don't quite remember why I did it like this. You're correct though, we shouldn't clobber `src_configure`. I could send an updated version sometime this week - sadly don't have much free time at the moment.